### ISSUE TITLE
Move repo to GitLab

### ISSUE THREAD

**Issue by iSWORD** [Microsoft/vscode#51113]
oh.. nevermind.

**Comment by seraku24** [comment]
I wonder how long this issue will stay open.  Here is hoping the VS Code team and contributors appreciate the humor and don't mind putting up with the memes.

**Comment by gusbemacbe** [comment]
Other Linux neo-Nazis and zealots of Dormammnu.

Fuck you, @iSWORD  and @seraku24 

Please, get out of Github, shitty Linux zealots. Less Linux users on Github, better. Nobody likes Linux community.

**Comment by iSWORD** [comment]
Oh did I hurt your feelings @gusbemacbe.. hmmm.. i wonder who created git..

![1ae](https://user-images.githubusercontent.com/1176779/40972430-fad8044c-68c0-11e8-9576-33401067e895.png)

**Comment by gusbemacbe** [comment]
You did not hurt my feeling, dear idiot Muslim.

**Comment by gusbemacbe** [comment]
Ah, @lolPants, @permyakovsv and @TheSilentHorizon created the fake accounts to attack GitHub. Pro-Russia Linux trolls.

**Comment by AlexHaber** [comment]
@gusbemacbe I would recommend that you stop being toxic. You are most likely violating Github ToS.

**Comment by gusbemacbe** [comment]
@AlexHaber , tell it to Muhammed so he also violated Github ToS. He published this topic to troll and harass us.

**Comment by seraku24** [comment]
Oh, I see @gusbemacbe's posts (and account?) were removed already.  But even so, I still feel I should respond to their comment about being a "Linux zealot."

I have been, for the most part, a happy consumer of Microsoft's products as well as other proprietary software vendors for decades; during which I have also spent a comparable, though lesser, time with Linux and the FLOSS side of life.  From my perspective, neither gets it all right, but similarly neither gets it all wrong either.  But, any programmer paying attention the past few years would see that Microsoft has made many leaps and bounds in the open-source world.  And while it is reasonable to question the future state of things when a company or service gets bought by or merged with another, I am not overly concerned at this stage.

All that said, I am not going to stop anyone from jumping ship; and I do think there is good humor in this whole situation.  Of course, I apologize if my comments and participation are seen as inappropriate.
